src/components/Email.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import './Email.scss';
+
+type EmailProps = {
+    isValid: boolean;
```

Comment:
_isVailid_ must be required prop, because by default the component will receive _isValid: undefined_- and this is bad practice. Component shold work with minimum props.

Actually, you have 3 variants here:

1. use _isInvalid_ prop and leave it to be not required. By default _isInvalid= undefined_ and component works
2. use _isValid_ and make it required.
3. use _isValid_ and default value when _isValid: undefined_ -
 _const Email = ({ isValid = false }: EmailProps) => {_